modules/gridmap/editor/grid_map_editor_plugin.cpp

```diff
@@ -674,7 +674,7 @@ void GridMapEditor::_show_viewports_transform_gizmo(bool p_value) {
 }
 
 EditorPlugin::AfterGUIInput GridMapEditor::forward_spatial_input_event(Camera3D *p_camera, const Ref<InputEvent> &p_event) {
-	if (!node) {
+	if (!node || Input::get_singleton()->get_mouse_mode() == Input::MOUSE_MODE_CAPTURED) {
```

Comment:
```suggestion
	// If the mouse is currently captured, we are most likely in freelook mode.
	// In this case, disable shortcuts to avoid conflicts with freelook navigation.
	if (!node || Input::get_singleton()->get_mouse_mode() == Input::MOUSE_MODE_CAPTURED) {
```